### ISSUE TITLE
selectedDate not working

### ISSUE THREAD

**Issue by Eschon** [priyank-p/simplepicker#37]
Im trying to set the initially selected date of the Picker, but it doesn't seem to work.

```JavaScript
const fromDate = new Date(Date.now() - (7 * 24 * 60 * 60 * 1000));

const pickerFrom = new SimplePicker(pickerContainerFrom, {
  selectedDate: fromDate,
});
const pickerTo = new SimplePicker(pickerContainerTo);
```

I've also tried it with only one picker to make sure the error is not caused by my multiple Picker setup.

```JavaScript
const fromDate = new Date(Date.now() - (7 * 24 * 60 * 60 * 1000));

const pickerFrom = new SimplePicker({
  selectedDate: fromDate,
});
```

**Comment by priyank-p** [comment]
Yeah, indeed this isn't working as expected.

**Comment by rohiturane** [comment]
I am getting the same issue. Did you find any solution?

**Comment by priyank-p** [comment]
Sorry for the delay but this is fixed. I will create a new release with the fix shortly.

**Comment by priyank-p** [comment]
This should be fixed in the new [v2.0.4 release](https://github.com/priyank-p/simplepicker/releases/tag/v2.0.4)
